Import/Esri/ArcGISRuntime/Toolkit/Controls/QmlApi/TimeSliderController.qml

```diff
@@ -18,26 +18,84 @@ import QtQuick 2.4
 import Esri.ArcGISRuntime 100.3
 
 /*!
-    \qmltype TimeSlider
+    \qmltype TimeSliderController
     \ingroup ArcGISQtToolkit
     \ingroup ArcGISQtToolkitQmlApi
     \inqmlmodule Esri.ArcGISRuntime.Toolkit.Controls
     \since Esri.ArcGISRutime 100.3
-    \brief
+    \brief The controller for the Time Slider tool.
+
+    The controller presents the temporal range of the data as a number of steps.
+    These steps allow the temporal extent to be set and animated by stepping through
+    the range.
 */
 Item {
     id: controller
 
+    /*!
+      \qmlproperty GeoView geoView
+      \brief the GeoView for this tool to \a geoView.
+
+      The view should be either a MapView or a SceneView.
```

Comment:
As a developer, do I need to set the GeoView on the TimeSlider & the Controller? Or does this automatically use what is set in the TimeSlider? Some doc to clarify this workflow would help